src/types/mod.rs

```diff
@@ -1,2 +1,33 @@
+use ethers::types::Address;
+use reth_db::table::{Decode, Encode};
+
 pub mod reputation;
 pub mod user_operation;
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
+pub struct WrapAddress(Address);
```

Comment:
I would move this from `mod.rs` to separate file, e.g., `src/types/utils.rs` or something like that.

All WrapAddress things.